### PR TITLE
Feature/dx 479 other features

### PR DESCRIPTION
**Pull request by NullableInt**
This pull request contains swish other features section with some neat updates and such.

### CODE REVIEW

File: _includes/subsite.md [hunk 1]
```diff
@@ -1,442 +1 @@
...
-    {% assign payeeReferenceLength = "30" %}
-{% elsif instrument=="mobilepay" %}
-    {% assign payeeReferenceLength = "50" %}
-{% elsif instrument=="checkout" %}
-    {% assign payeeReferenceLength = "30" %}
-{% else %}
-    {% assign payeeReferenceLength = "30" %}
-{% endif %}
-
 ## Subsite
```
Comment (asbjornu) [inline _includes/subsite.md]: Is this file useful anymore if all it contains is a header?

Comment (NullableInt) [reply _includes/subsite.md]: Probably not. I can delete it

File: payments/swish/other-features.md [hunk 2]
```diff
@@ -18,20 +18,32 @@ sidebar:
...
 
+### Payee reference
+
+{% include payee-info.md %}
+
+{% include expand-parameter.md %}
+
+#### Transaction
+
+{% include transactions-reference.md %}
```
Comment (trieutran91) [inline payments/swish/other-features.md]: Both transaction.md and transactions.md (inside of this include, transactions-reference.md) have URI's with `creditcard` value. Maybe add viable tags that can allows us to change the values based on instrument?

File: payments/swish/other-features.md [hunk 3]
```diff
@@ -18,20 +18,32 @@ sidebar:
...
 
-{% include recurring-card-payments.md %}
+{% include settlement-reconciliation.md %}
 
-{% include subsite.md payment-instrument="swish" %}
+{% include payment-link.md %}
 
 ### Prices
 
 {% include prices.md %}
```
Comment (trieutran91) [inline payments/swish/other-features.md]: This include has URI's with `creditcard` value. Maybe add variable tags that can allows us to change the values based on instrument?

**Comment by NullableInt** [comment]
> I think that a Create Payment section is missing. The Purchase operation is in use. Maybe recur and verify too?

Create payment is in the `after payment` section in other instruments.

**Comment by trieutran91** [comment]
> > I think that a Create Payment section is missing. The Purchase operation is in use. Maybe recur and verify too?
> 
> Create payment is in the `after payment` section in other instruments.

Oh, I found Create Payment in `Other Features` in both Card and Invoice.
Also, would it be misplaced in `After Payment`?

**Comment by NullableInt** [comment]
> > > I think that a Create Payment section is missing. The Purchase operation is in use. Maybe recur and verify too?
> > 
> > 
> > Create payment is in the `after payment` section in other instruments.
> 
> Oh, I found Create Payment in `Other Features` in both Card and Invoice.
> Also, would it be misplaced in `After Payment`?

No idea. It does not belong in either place.

File: payments/direct-debit/other-features.md [hunk 4]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+
+{
+    "payment": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
+    "sales": {
+        "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales",
+        "salesList": [
+            {
+                "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales/12345678-1234-1234-1234-123456789012",
+                "selectedBank": "NordeaFI",
+                "deviceIsMobile": "TRUE|FALSE",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]: No pipes in code examples, please. Possible values should be described in the property table below the code example.

```suggestion
                "deviceIsMobile": false,
```

File: payments/direct-debit/other-features.md [hunk 5]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+                "transaction": {
+                    "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions/12345678-1234-1234-1234-123456789012",
+                    "created": "2018-09-14T01:01:01.01Z",
+                    "updated": "2018-09-14T01:01:01.03Z",
+                    "type": "Sale",
+                    "state": "Initialized|Completed|Failed",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
                    "state": "Initialized",
```

File: payments/direct-debit/other-features.md [hunk 6]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+                    "number": 1234567890,
+                    "amount": 1000,
+                    "vatAmount": 250,
+                    "description": "Test transaction",
+                    "payeeReference": "AH123456",
+                    "failedReason": "",
+                    "failedActivityName": "",
+                    "failedErrorCode": "",
+                    "failedErrorDescription": "",
+                    "isOperational": "TRUE|FALSE",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
                    "isOperational": true,
```

File: payments/direct-debit/other-features.md [hunk 7]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+                    "operations": []
+                }
+            },
+            {
+                "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales/12345678-1234-1234-1234-123456789013",
+                "selectedBank": "NordeaFI|...",
+                "deviceIsMobile": "TRUE|FALSE",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
                "deviceIsMobile": false,
```

File: payments/direct-debit/other-features.md [hunk 8]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+                    "number": 1234567890,
+                    "amount": 1000,
+                    "vatAmount": 250,
+                    "description": "Test transaction",
+                    "payeeReference": "AH123456",
+                    "failedReason": "",
+                    "failedActivityName": "",
+                    "failedErrorCode": "",
+                    "failedErrorDescription": "",
+                    "isOperational": "TRUE|FALSE",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
                    "isOperational": true,
```

File: payments/direct-debit/other-features.md [hunk 9]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+            {
+                "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/sales/12345678-1234-1234-1234-123456789013",
+                "selectedBank": "NordeaFI|...",
+                "deviceIsMobile": "TRUE|FALSE",
+                "transaction": {
+                    "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions/12345678-1234-1234-1234-123456789013",
+                    "created": "2018-09-14T01:01:01.01Z",
+                    "updated": "2018-09-14T01:01:01.03Z",
+                    "type": "Sale",
+                    "state": "Initialized|Completed|Failed",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
                    "state": "Initialized",
```

File: payments/direct-debit/other-features.md [hunk 10]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+        "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/reversals",
+        "reversalList": [
+            {
+                "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/reversals/12345678-1234-1234-1234-123456789012",
+                "transaction": {
+                    "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions/12345678-1234-1234-1234-123456789012",
+                    "created": "2016-09-14T01:01:01.01Z",
+                    "updated": "2016-09-14T01:01:01.03Z",
+                    "type": "Reversal",
+                    "state": "Initialized|Completed|Failed",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
                    "state": "Initialized",
```

File: payments/direct-debit/other-features.md [hunk 11]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+                    "number": 1234567890,
+                    "amount": 1000,
+                    "vatAmount": 250,
+                    "description": "Test transaction",
+                    "payeeReference": "AH123456",
+                    "isOperational": "TRUE|FALSE",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
                    "isOperational": true,
```

File: payments/direct-debit/other-features.md [hunk 12]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+{
+    "payment": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
+    "reversal": {
+        "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/reversals/12345678-1234-1234-1234-123456789012",
+        "transaction": {
+            "id": "/psp/directdebit/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions/12345678-1234-1234-1234-123456789012",
+            "created": "2016-09-14T01:01:01.01Z",
+            "updated": "2016-09-14T01:01:01.03Z",
+            "type": "Reversal",
+            "state": "Initialized|Completed|Failed",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
            "state": "Initialized",
```

File: payments/direct-debit/other-features.md [hunk 13]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+            "number": 1234567890,
+            "amount": 1000,
+            "vatAmount": 250,
+            "description": "Test transaction",
+            "payeeReference": "AH123456",
+            "isOperational": "TRUE|FALSE",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
            "isOperational": true,
```

File: payments/swish/other-features.md [hunk 14]
```diff
@@ -73,14 +85,5 @@ All Swish error types will have the following URI in front of type:
 | *swishdeclined*      | 403    | DS24                 | Payment timed out (Bank didn't respond).                                                        |
 | *systemerror*        | 500    | Any other error code |
```
Comment (asbjornu) [inline payments/swish/other-features.md]: Code format all values in this table, please.

**Comment by trieutran91** [comment]
> > > > I think that a Create Payment section is missing. The Purchase operation is in use. Maybe recur and verify too?
> > > 
> > > 
> > > Create payment is in the `after payment` section in other instruments.
> > 
> > 
> > Oh, I found Create Payment in `Other Features` in both Card and Invoice.
> > Also, would it be misplaced in `After Payment`?
> 
> No idea. It does not belong in either place.

I talked with David R. It seems like Purchase is the only available operation atm :)

File: payments/direct-debit/other-features.md [hunk 15]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+                    "created": "2016-09-14T01:01:01.01Z",
+                    "updated": "2016-09-14T01:01:01.03Z",
+                    "type": "Reversal",
+                    "state": "Initialized",
+                    "number": 1234567890,
+                    "amount": 1000,
+                    "vatAmount": 250,
+                    "description": "Test transaction",
+                    "payeeReference": "AH123456",
+                    "isOperational": "TRUE",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
                    "isOperational": true,
```

File: payments/direct-debit/other-features.md [hunk 16]
```diff
@@ -32,6 +32,427 @@ sidebar:
...
+            "created": "2016-09-14T01:01:01.01Z",
+            "updated": "2016-09-14T01:01:01.03Z",
+            "type": "Reversal",
+            "state": "Initialized",
+            "number": 1234567890,
+            "amount": 1000,
+            "vatAmount": 250,
+            "description": "Test transaction",
+            "payeeReference": "AH123456",
+            "isOperational": "TRUE",
```
Comment (asbjornu) [inline payments/direct-debit/other-features.md]:
```suggestion
            "isOperational": true,
```

**Pull request merged**